Editor/CoreAPI/GameLiftComputeManager.cs

```diff
@@ -61,9 +61,9 @@ public GameLiftComputeManager(CoreApi coreApi, IAmazonGameLiftWrapper wrapper)
             }
             catch (Exception ex)
             {
-                var errorBox = _container.Q<VisualElement>("ComputeErrorInfoBox");
-                errorBox.style.display = DisplayStyle.Flex;
-                errorBox.Q<Label>().text = ex.Message;
+                // var errorBox = _container.Q<VisualElement>("ComputeErrorInfoBox");
+                // errorBox.style.display = DisplayStyle.Flex;
+                // errorBox.Q<Label>().text = ex.Message;
```

Comment:
Why are we commenting this out?

Reply:
Because the error box was not fully implemented. This will be addressed in the follow up PR to #96